src/test/java/org/phoebus/channelfinder/ChannelRepositoryIT.java

```diff
@@ -180,39 +183,39 @@ public void testChannelsExist() {
         assertTrue("Failed to check the non-existance of 'non-existant-channel'", !channelRepository.existsByIds(Arrays.asList("test-channel1", "non-existant-channel")));    
     }
 
-    /**
-     * find all channels
-     */
-    @Test
-    public void findAllXmlChannels() {
-        XmlChannel testChannel = new XmlChannel("testChannel","testOwner",testProperties,testTags);
-        XmlChannel testChannel1 = new XmlChannel("testChannel1","testOwner1",testProperties,testTags);
-        List<XmlChannel> testChannels = Arrays.asList(testChannel, testChannel1);
-        Set<XmlChannel> createdChannels = Sets.newHashSet(channelRepository.indexAll(testChannels));
-        cleanupTestChannels = Arrays.asList(testChannel,testChannel1);
-
-        try {
-            //Thread.sleep(2000);
-            Set<XmlChannel> listedChannels = Sets.newHashSet(channelRepository.findAll());
-            // verify the channel was created as expected
-            //assertEquals("Failed to list all created channel", createdChannels, listedChannels);
-            assertTrue("Failed to list all created channel", listedChannels.containsAll(createdChannels));
-        } catch (Exception e) {
-            e.printStackTrace();
-        } 
-    }
+//    /**
+//     * find all channels
+//     */
+//    @Test
+//    public void findAllXmlChannels() {
+//        Channel testChannel = new Channel("testChannel","testOwner",testProperties,testTags);
+//        Channel testChannel1 = new Channel("testChannel1","testOwner1",testProperties,testTags);
```

Comment:
Why is this now commented out?

Reply:
The final all had been initially added to satisfy the spring `CrudRepository ` interface.. however, for CF where there might be millions of channels this operation is not necessarily useful. Hence it is now not supported.

```
    @Override
    public Iterable<Channel> findAll() {
        throw new UnsupportedOperationException(TextUtil.FIND_ALL_CHANNELS_NOT_SUPPORTED);
    }
```

Reply:
removed